code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -13,10 +13,7 @@
 	if(isliving(target))
 		var/mob/living/L = target
 		if(L.reagents)
-			L.reagents.add_reagent("toxin", poison_per_bite)
-			if(prob(poison_per_bite))
-				L << "<span class='danger'>You feel a tiny prick.</span>"

```

Comment:
Do not remove the tiny prick! "tiny prick" is a fantastic meme and warning that something bad *might* happen to you!